crates/rnote-engine/src/pens/typewriter/penevents.rs

```diff
@@ -454,15 +454,22 @@ impl Typewriter {
                 pen_down,
                 ..
             } => {
-                // detect hover state
-                *modify_state = if typewriter_bounds
-                    .map(|b| b.contains_local_point(&element.pos.into()))
-                    .unwrap_or(false)
-                {
-                    ModifyState::Hover(element.pos)
-                } else {
-                    ModifyState::Up
-                };
+                match modify_state {
```

Comment:
Wouldn't it be nicer to write something like `if !matches!(ModifyState::Selecting {..})` ?

Reply:
Fixed